### ISSUE TITLE
Indicate dependency in manifest

### ISSUE THREAD

**Issue by neoeinstein** [TheFrett/msfs_g36_project#40]
Rather than relying on the implicit ordering of alphabetical order, you may be able to use the `dependencies` property of the manifest to indicate that this mod depends on the G1000 mod. That should fix the ordering issues and make it so that you don't need the z- hack.

**Comment by TheFrett** [comment]
I didn't know about this and will certainly test it, Thanks!

**Comment by TheFrett** [comment]
How exactly should this work?

I added:

"dependencies": [
	{
	  "name": "workingtitle-g1000",
	  "package_version": "0.2.0"
    },
    {
      "name": "fs-base-propdefs",
      "package_version": "0.1.2"
	}
  ],

(this refers to the v0.3 package of the G1000, they forgot to update their package version number). I removed the 'z-hack' but to no succes. The wrong WTEngineDisplay.xml is loaded (the one that ships with the G1000 mod), which I can tell by the wrong MFD electrical system indications.

What should I do?
